### PR TITLE
implement Genre class

### PR DESCRIPTION
**Pull request by Balikuddembe**
Hi @team,
In this PR,I have implemented the following
* Create Genre class with an association to the Item class (in a separate .rb file).
* All Genre class properties visible in the diagram should be defined and set up in the constructor method.

Implement methods:
* add_item method in the Genre class
* should take an instance of the Item class as an input
* should add the input item to the collection of items
* should add self as a property of the item object (by using the correct setter from the item object)

### CODE REVIEW

File: src/associations/genre.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require_relative '../items/item'
+class Genre
+  attr_accessor :name, :items
+  attr_reader :id, :items
```
Comment (lucascomamala) [inline src/associations/genre.rb:3]: You duplicated the :items instance variable, we only need it once.

**Review by lucascomamala — CHANGES_REQUESTED** [review]
Hi @Balikuddembe , 

Good job so far!
There is one issue that needs to be fixed before approval

## Highlights
Well documented :heavy_check_mark: 
gitflow :heavy_check_mark: 
Correct initialization :heavy_check_mark: 
Correct add_item method :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

File: src/associations/genre.rb (hunk 1 above)
Comment (Balikuddembe) [reply src/associations/genre.rb:3]: Thank you @lucascomamala for the review comment.I will go ahead to implement the change.

**Review by lucascomamala — APPROVED** [review]
Hi @Balikuddembe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

**Pull request merged**